modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -499,7 +499,9 @@ Error GDScriptWorkspace::parse_local_script(const String &p_path) {
 
 String GDScriptWorkspace::get_file_path(const String &p_uri) const {
 	String path = p_uri;
-	path = path.replace(root_uri + "/", "res://");
+	path = (path.matchn("*///*")) ? path.replace("///", "//") : path;
+	path = (path.matchn("*%3A*")) ? path.replace("%3A", ":") : path;
```

Comment:
Is the actually match check useful? You should be able to just run `replace()` directly and if there's nothing to replace, it won't do anything.

Reply:
Aaah could be depending on what operation is faster though I'm guessing replace does some sort of match as well to check if there is anything to replace, i can remove that now and recommit.

Reply:
done removed match checks.